src/UnoDrive.Shared/ViewModels/MyFilesViewModel.cs

```diff
@@ -83,40 +84,46 @@ public bool IsStatusBarLoading
 
 		public async void ItemClick(object sender, ItemClickEventArgs args)
 		{
-			if (args.ClickedItem is OneDriveItem driveItem)
+			if (args.ClickedItem is not OneDriveItem driveItem)
+				return;
+			
+			if (driveItem.Type == OneDriveItemType.Folder)
 			{
-				if (driveItem.Type == OneDriveItemType.Folder)
+				try
```

Comment:
since we are in an `async void` is good to wrap our `await` calls inside a try/catch block